### ISSUE TITLE
Refbox Configuration Message

### ISSUE THREAD

**Issue by pkohout** [robocup-logistics/rcll-refbox#193]
Hi,

as far as I remember we had some discussion about a message that contains the field size, so that this config can be done automatically by the teams. I had some discussion and I think this would be a good message in the Setup phase were we can also put other information like:

- Game Field Size
- Challenges enabled
- Machine to Aruco Tag ID

for us this would eliminate a lot of the manual changes we have do to right now if we switch the modes/sizes in the refbox. The Tag ID mapping is because we already have it in multiple times hardcoded in different languages :D.

**Comment by TarikViehmann** [comment]
Good idea, similar to https://github.com/robocup-logistics/rcll-refbox/issues/154. I can prepare a branch for that for testing.
Maybe we should send the tag info with the machines instead.
